### DISCUSSION TITLE
I/O delay

### DISCUSSION THREAD

**Discussion by To0wnn** [dougiteixeira/proxmoxve#268] [Feature request] [answered]
Hi,

Can IO Delay be added to the sensors for the Node?

Kind regards,

Tony

**Comment by jeremysherriff** [comment]
Check #317 and perhaps vote on it?

**Comment by mZ738** [comment, marked as answer]
`IO delay` is a node sensor since 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)), on by default — what the node summary in Proxmox shows as IO delay. Closing as done.
